ruby-training/trainees/david/test_katas/08/lib/playing_card_deck.rb

```diff
@@ -4,10 +4,23 @@
 class PlayingCardDeck < Deck
   def initialize
     super
-    # your code here
+    @ranks = [*(2..10), 'J', 'Q', 'K', 'A']
```

Comment:
`@ranks` and `@suits` are only used once during initialization and they don't change (even accross instances). Consider refactoring them out into constants instead of instance variables. There's no need for the `PlayingCardDeck` object to store them.